client/components/Platforms.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const PlatformsStyled = styled.div`
+  width: 338.91px;
+  height: 22.31px;
+`;
```

Comment:
the images from platforms are in png format, and they are all 24px x 24px .  That can be tweeked as necessary either on your end or mine.
the 24px x 24px includes some padding around the actual 'icon' shape.

Reply:
OK ty for the guidance ... and the padding :).